lib/fs.js

```diff
@@ -43,6 +43,18 @@ const internalUtil = require('internal/util');
 const assertEncoding = internalFS.assertEncoding;
 const stringToFlags = internalFS.stringToFlags;
 const getPathFromURL = internalURL.getPathFromURL;
+const O_RDONLY = constants.O_RDONLY | 0;
```

Comment:
I just copied these verbatim from lib/internal/fs.js. It implicitly converts the value to a 32-bit integer, so there may be a reason for that, performance or otherwise?